### PR TITLE
[4.0] Remove module class suffix from module files

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

Removes module class suffix uses from module files. Suffix should now be handled by module chromes, see #17447.

### Testing Instructions

Code review.

### Documentation Changes Required

IDK.

### CODE REVIEW

File: administrator/modules/mod_feed/tmpl/default.php [hunk 1]
```diff
@@ -65,7 +65,7 @@
 		$iUrl   = $feed->image ?? null;
 		$iTitle = $feed->imagetitle ?? null;
 		?>
-		<div style="direction: <?php echo $rssrtl ? 'rtl' :'ltr'; ?>; text-align: <?php echo $rssrtl ? 'right' :'left'; ?> !important" class="feed<?php echo $moduleclass_sfx; ?>">
+		<div style="direction: <?php echo $rssrtl ? 'rtl' :'ltr'; ?>; text-align: <?php echo $rssrtl ? 'right' :'left'; ?> !important" class="feed">
```
Comment (Quy) [inline administrator/modules/mod_feed/tmpl/default.php]:
```suggestion
		<div style="direction: <?php echo $rssrtl ? 'rtl' : 'ltr'; ?>; text-align: <?php echo $rssrtl ? 'right' : 'left'; ?> !important" class="feed">
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c12a8ce5bb777704e9e7b009c13abd66a3266353<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25322">issues.joomla.org/tracker/joomla-cms/25322</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on c12a8ce5bb777704e9e7b009c13abd66a3266353<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25322">issues.joomla.org/tracker/joomla-cms/25322</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25322">issues.joomla.org/tracker/joomla-cms/25322</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**